Library/Formula/unbound.rb

```diff
@@ -19,13 +19,10 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-libevent=#{Formula["libevent"].opt_prefix}",
                           "--with-ssl=#{Formula["openssl"].opt_prefix}"
+    inreplace "doc/example.conf", 'username: "unbound"', "username: \"#{ENV["USER"]}\""
```

Comment:
This won't work unfortunately because it needs to use the user at `post_install` rather than bottling time.